### ISSUE TITLE
Account Asset Requests

### ISSUE THREAD

**Issue by Fraccas** [JACK-COM/reachduck#37]
We've recently started using ReachDuck for persistent wallet login and we've noticed that when we use "reconnectUser()" or "connectUser()" it seems to try to query every asset from the logged wallet with most of the requests failing. These wallets have hundreds and sometimes thousands of assets in them. 

Is there anything we can do to prevent these requests?

**Comment by Fraccas** [comment]
Digging through the src code I believe there needs to be an option for excluding certain information when pulling the account with lookupAccountByID(addr)

```
/**
 * Lookup an Algorand account by its address
 * @param addr Account address
 * @returns Algorand account information
 */
export async function fetchAccount(addr: string) {
  const Indexer = useIndexerClient();
  const result: any = await Indexer.lookupAccountByID(addr)
    .includeAll(false) // added
    .exclude("all") // added
    .do()
    .catch(fallbackAcct);
  return result.account;
}
```

**Comment by MrJackdaw** [comment]
@Fraccas Thanks for reporting this; I published `v0.3.0` tonight and included your suggestions. The library will default to fetching a maximum of `10` assets, though you can 
* specify a different limit
* skip asset fetching, and/or
* skip balance fetching
by providing relevant options to [`connectUser`](https://jack-com.github.io/reachduck/stdlib_functions.html#connectuser). or `reconnectUser`. The docs have also been updated. 

Please test when you can and let me know if it improved (or I introduced any new bugs). If it is all good, please let me know and close this issue. 

Cheers!

**Comment by Fraccas** [comment]
> 

Works great thanks!
